packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/InviteCollaborator/RecipientContainer.spec.ts

```diff
@@ -1,70 +1,39 @@
-import { createLocalVue, mount } from '@vue/test-utils'
```

Comment:
I just converted this file to `TypeScript` while preserving the "logic" of the tests. I'm honestly not sure what the component does... also, the avatar-check looks broken to me (in the component itself, not in the tests).

Reply:
The component is the small item within the sharee invite form, that represents a selected sharee within the input field.

Reply:
Which check do you mean? The `hasAvatar` within `data`? Mainly it looks wrong to me that it's in `data` 😅

Reply:
Yeah that's what I meant :D `this.hasAvatar` will never evaluate to `true` because it sits in `this.formattedRecipient.hasAvatar`.

Reply:
Probably never catched somebodies eye because ocis doesn't have avatars 😆